### PR TITLE
add more information for matched cards

### PR DESCRIPTION
**Pull request by abcotter**
In this ticket:

- Added info pulled from the API this will allow us to display Artist details and art information
- displayed the art details when a match occurs

Test:
- ran locally to ensure that the info was pulled correctly and displayed as expected
- ran a local production instance to ensure no errors occurred in prod

**NOTE: when we select the art pieces to build out the art object dictionary, we need to ensure they have the information formatted and stored!** 

        "artName": artDetails["title"],
        "artUrl": artDetails["primaryImage"],
        "artistName": artDetails["artistDisplayName"],
        "artCulture": artDetails["culture"],
        "artPeriod": artDetails["period"],
        "birthYear": artDetails["artistBeginDate"],
        "deathYear": artDetails["artistEndDate"],

### CODE REVIEW

**Comment by FatimahAreola** [comment]
This code is pending review because I would like to test the requirements for artist details against the code to query the MET API. **No action required from developer.** PR will be approved pending successful merge and test with GenerateNewArt branch.

**Comment by FatimahAreola** [comment]
@abcotter Some of the fields added return blank or null value for any artist I test with. Specifically the field "art period" does not return a value for any artist and "art culture" often does not return a value. This is understandable, and to be expected for certain fields given the nature of such an open-source program. How does the front-end handle the display of such cases? I think it would be beneficial to return a value like "Unknown" or "Unavailable" for these things. If there are other similar fields in the MET API that do return values more frequently, maybe we should replace with those.

**Comment by abcotter** [comment]
@FatimahAreola, if the information is missing it just is left blank, I can also remove these fields? let me know what you think

File: frontend/frontend/src/components/MatchPopup.vue
Comment (FatimahAreola) [inline frontend/frontend/src/components/MatchPopup.vue]:
```suggestion
```
I would remove this value as it comes up blank for all artists

File: server/app.py
Comment (FatimahAreola) [inline server/app.py]:
```suggestion
```
recommending removal as appears blank for all art

File: server/app.py
Comment (FatimahAreola) [inline server/app.py]:
```suggestion
```
recommending removal as appears blank for all artists

**Review by FatimahAreola — CHANGES_REQUESTED** [review]
I've requested the removal of the art period since I don't believe it makes sense from a UX perspective to have a field that will always appear blank for our specific game. I will leave the option of culture up to you as a design decision and have provided guidelines on how to test how each artist appears to you offline.

Thanks Abi :)

**Comment by FatimahAreola** [comment]
The code looks good to me but I am getting an error while testing, it appears that one of the art pieces currently as a test on our main branch (ID 435882) has been taken down from the MET API. I don't believe this should be an issue moving forward as we are changing the Object IDs this sprint but wanted to surface this issue. Would you mind testing on your end to make sure this is the only issue? I'm also assigning @m2qaderi to ensure she can test the branch on her end since this is a little strange.


Edit: Tested again and I retract my prior comment but still need to figure out the issues I'm having with testing.

**Review by m2qaderi — APPROVED** [review]
I tested and it works fine for me!

**Review by FatimahAreola — CHANGES_REQUESTED** [review]
Was able to test, everything looks great! Please merge from main before I can approve and then you should be good to merge.

**Review by FatimahAreola — APPROVED** [review]

**Pull request merged**
